test/cljc/snippets/math/random.cljc

```diff
@@ -42,3 +42,14 @@
   (q/fill 0)
   (q/random-seed 42)
   (q/text (str "(q/random 42) = " (q/random 42)) 10 20))
+
+(defsnippet random-2d {}
+  (q/background 255)
+  (q/fill 0)
+  (q/text (str "(q/random-2d) = " (q/random-2d)) 10 20))
```

Comment:
Yes, it's a good idea. Historically all tests here are "visual" and even the math-related ones don't check if results are correct. But It wouldn't hurt to add such checks into visual tests.